libraries/scripting/dependencies/src/kotlin/script/experimental/dependencies/ExternalDependenciesResolver.kt

```diff
@@ -16,7 +17,7 @@ interface ExternalDependenciesResolver {
     fun acceptsRepository(repositoryCoordinates: RepositoryCoordinates): Boolean
     fun acceptsArtifact(artifactCoordinates: String): Boolean
 
-    suspend fun resolve(artifactCoordinates: String): ResultWithDiagnostics<List<File>>
+    suspend fun resolve(artifactCoordinates: String, location: SourceCode.Location? = null): ResultWithDiagnostics<List<File>>
```

Comment:
I'd name the parameter `sourceCodeLocation` - here it is not obvious from the context.

Reply:
✅